src/Interactive/Host/Interactive/Core/InteractiveHost.RemoteAsyncOperation.cs

```diff
@@ -67,7 +64,7 @@ public void Completed(TResult result)
 
             private void ProcessExited()
             {
-                Completed(result: default(TResult));
+                Completed(result: default!);
             }
 
             public void SetResult(TResult result)
```

Comment:
can we annotate this (ie.. [MaybeNull]) to make it so you don't need `!`?

Reply:
`[MaybeNull]` doesn't work :(